FrostyModManager/Windows/MainWindow.xaml.cs

```diff
@@ -981,14 +981,19 @@ private void largeScreenshot_MouseDown(object sender, MouseButtonEventArgs e)
 
         private void FrostyWindow_Drop(object sender, DragEventArgs e)
         {
-            if (e.Data.GetDataPresent(DataFormats.FileDrop, true) == true)
+            if (e.Data.GetDataPresent(DataFormats.FileDrop, true))
             {
                 string[] filenames = (string[])e.Data.GetData(DataFormats.FileDrop, true);
                 InstallMods(filenames);
 
                 ICollectionView view = CollectionViewSource.GetDefaultView(availableModsList.ItemsSource);
                 view.Refresh();
             }
+            else if (e.Data.GetFormats().Any(f => f == "FileContents"))
```

Comment:
does that not write them to the tmp storage and then u should be able to import them anyways?

Reply:
windows zip doesnt, winrar however does and it will satisfy the first if statement